src/pages/Overview/OverviewPage.tsx

```diff
@@ -3,9 +3,10 @@
  *
  * Component that renders the overview page content in the dashboard.
  */
-
-import { Typography } from '@mui/material';
+import { OverviewImageListContainer } from '@containers/OverviewImageList';
 
 export const OverviewPage = () => (
-    <Typography variant="h1">Overview Page</Typography>
+    <>
+        <OverviewImageListContainer />
+    </>
```

Comment:
Either the container can be removed, or this can be removed. What purpose does a React component solves, if it just imports another, and wraps in inside fragment, and exports? The container is useless. Its logic must live in the page, and container should not exist. Same goes for all other containers. The page is the container.
avoid over abstraction